test/sequential/test-http2-session-timeout.js

```diff
@@ -34,12 +34,14 @@ server.listen(0, common.mustCall(() => {
     request.end();
 
     request.on('end', () => {
-      if (attempts) {
-        setTimeout(() => makeReq(attempts - 1), callTimeout);
+      const diff = process.hrtime(startTime);
+      const milliseconds = (diff[0] * 1e3 + diff[1] / 1e6);
+      if (milliseconds < serverTimeout * 2) {
+        setTimeout(() => makeReq(), callTimeout);
```

Comment:
Nit: if I'm not wrong we can pass `makeReq` directly without wrapping it in another function.

Reply:
Indeed. Changed. Thanks!